qiskit_experiments/data_processing/data_processor.py

```diff
@@ -54,23 +54,29 @@ def append(self, node: DataAction):
         """
         self._nodes.append(node)
 
-    def __call__(self, datum: Dict[str, Any]) -> Any:
+    @property
+    def is_trained(self) -> bool:
```

Comment:
Same here, do all Data processors require training? Seems specific only to ML type ones.

Reply:
As discussed, I introduced a TrainableDataAction see here: 74e928b